log/log.go

```diff
@@ -46,3 +46,9 @@ func Trace(msg string, ctx ...interface{}) {
 	metrics.GetOrRegisterCounter("trace", nil).Inc(1)
 	l.Output(msg, l.LvlTrace, CallDepth, ctx...)
 }
+
+// GetHandler return the Handler assigned to root
+func GetHandler() l.Handler {
+	lh := l.Root()
+	return lh.GetHandler()
```

Comment:
What does a handler exactly define and do?

Reply:
Once the logger is initialized it uses a handler that is passed on to the ethereum logger.
the problem is with visibility from the swap log side.

It did not have a method to get the logger handler method.
So I extended its functionality to expose the inner handler.